deploy/template.yaml

```diff
@@ -0,0 +1,1352 @@
+#                Type: AWS::ElasticLoadBalancingV2::Listener
+#                LogicalID: ApplicationLoadBalancerListenerHTTPS
+#              TargetGroups:
+#                - ApplicationLoadBalancerTargetGroupBlue
+#                - ApplicationLoadBalancerTargetGroupGreen
+#    Type: AWS::CodeDeploy::BlueGreen
+
+Parameters:
+  Environment:
+    Description: The name of the environment to deploy to
+    Type: String
+    AllowedValues:
+      - "dev"
+      - "build"
+      - "staging"
+      - "integration"
+      - "production"
+  VpcStackName:
+    Description: The name of the stack that defines the VPC in which this container will run.
+    Type: String
+  RedisStackName:
+    Description: The name of the stack that defines the Redis cluster
+    Type: String
+    Default: shared-redis
+  ContainerPort:
+    Description: The port on which the container is running
+    Type: Number
+    Default: 6001
+  DevFrontendContainer:
+    Description: The ECR container for overriding in development, if you want to target a specific version/tag
+    Type: String
+    Default: 301577035144.dkr.ecr.eu-west-2.amazonaws.com/account-mgmt-frontend-image-containerrepository-aax5wa5iq29s:latest
+  ProductTagValue:
+    Description: Value for the Product Tag
+    Type: String
+    Default: GOV.UK Sign In
+  OwnerTagValue:
+    Description: Value for the Owner Tag
+    Type: String
+    Default: govuk-accounts-tech@digital.cabinet-office.gov.uk
+  SourceTagValue:
+    Description: Value for the Source Tag
+    Type: String
+    Default: alphagov/di-authentication-account-management/deploy/template.yaml
+  PermissionsBoundary:
+    Description: The ARN of the permissions boundary to apply when creating IAM roles
+    Type: String
+    Default: none
+
+Conditions:
+  IsNotDevelopment: !Or
+    - !Equals [ !Ref Environment, build ]
+    - !Equals [ !Ref Environment, staging ]
+    - !Equals [ !Ref Environment, integration ]
+    - !Equals [ !Ref Environment, production ]
+  IsProduction: !Equals [ !Ref Environment, production ]
+  UsePermissionsBoundary:
+    Fn::Not:
+      - Fn::Equals:
+          - !Ref PermissionsBoundary
+          - "none"
+
+Mappings:
+  # see https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+  ElasticLoadBalancerAccountIds:
+    eu-west-2:
+      AccountId: 652711504416
+  EnvironmentVariables:
+    ### These environment variables are referenced further down the file.
+    ### Please ensure that any variables defined for an environment are defined for _all_ environments.
+    dev:
+      NODEENV: "development"
+      APIBASEURL: "https://oidc.dev.account.gov.uk"
+      AMAPIBASEURL: "https://manage.dev.account.gov.uk"
+      BASEURL: "home.dev.account.gov.uk"
+      SESSIONEXPIRY: "7200000"
+      AMYOURACCOUNTURL: ""
+      GOVACCOUNTSPUBLISHINGAPIURL: "account-api.integration.publishing.service.gov.uk"
+      SUPPORTINTERNATIONALNUMBERS: "0"
+      SUPPORTLANGUAGECY: "0"
```

Comment:
```suggestion
      SUPPORTLANGUAGECY: "1"
```

Reply:
This needs to be on in all envs right now.

Reply:
done